tendrl/node_agent/message/logger.py

```diff
@@ -34,6 +34,11 @@ def __init__(self, message):
     def push_operation(self):
         NS.etcd_orm.client.write(
             "/Messages/jobs/%s" % self.message.job_id,
+            '',
+            dir=True,
+            ttl=NS.config.data['message_retention_time'])
+        NS.etcd_orm.client.write(
+            "/Messages/jobs/%s" % self.message.job_id,
```

Comment:
because of sequence id ttl is not work here, that why i am writing base dir first with ttl and then i am writing content with sequence id